core/src/timer.rs

```diff
@@ -41,7 +41,12 @@ impl<'gc> Timers<'gc> {
             return None;
         }
 
-        let level0 = context.stage.root_clip();
+        let level0 = if let Some(level0) = context.stage.root_clip() {
+            level0
+        } else {
+            tracing::warn!("Ignoring timers as there's no root movie");
+            return None;
```

Comment:
This should be gated behind `!context.is_actionscript_3() `, since we can still run Avm2 timers without a root movie

Reply:
Yeah realized that now and working on a fix!

Reply:
Fixed (but differently, I run the AVM2 timers only)